common/mount.py

```diff
@@ -153,26 +150,25 @@ def __init__(self,
             action = None
             running = cache.status()
             if not running:
-                logger.debug('pw-cache is not running', self)
-                action = 'start'
+                logger.debug("pw-cache is not running", self)
```

Comment:
Please use single `'` if possible. This is because of consistency. I know you will found enough exceptions of that. But that is also just outdated code we need to fix.

Anyway. Plesae don't blow up the PR. Don't change to much. Keep it simple please and keep other modifications for other PRs.

Reply:
ok. black autoformatter changed that. going to revert those changes